package/kernel/linux/modules/wpan.mk

```diff
@@ -106,6 +106,21 @@ endef
 
 $(eval $(call KernelPackage,cc2520))
 
+define KernelPackage/ca8210
+  SUBMENU:=$(WPAN_MENU)
+  TITLE:=Cascoda CA8210 transceiver driver
+  DEPENDS:=+kmod-mac802154 @!LINUX_3_18 @!LINUX_4_4 @!LINUX_4_9
+  KCONFIG:=CONFIG_IEEE802154_CA8210 \
+	CONFIG_IEEE802154_CA8210_DEBUGFS=n \
+	CONFIG_COMMON_CLK=y \
```

Comment:
Please remove this CONFIG_COMMON_CLK here this could cause problems with some targets. 
If a target does not define CONFIG_COMMON_CLK in their configuration the ca8210 will just not be build.